app/views/components/navigation_bar.scala.html

```diff
@@ -33,23 +32,23 @@
 
                 <ul class="moj-primary-navigation__list">
                     <li class="moj-primary-navigation__item">
-                        <a id="navigation-home-link" class="moj-primary-navigation__link" aria-current="page" href="@appConfig.emcsTfeHomeUrl">@messages("navigation.home")</a>
+                        <a id="navigation-home-link" class="moj-primary-navigation__link" @if(pageTitle == messages("viewMovement.title")) { aria-current="page" } href="@appConfig.emcsTfeHomeUrl">@messages("navigation.home")</a>
```

Comment:
Matching on page title can be risky, each page will have a different title. IMO I think having some kind of viewmodel object that we pass to this component would be better. Like so:
https://github.com/hmrc/emcs-tfe-frontend/commit/f43725b4a1c2276c1c4c7d88604c69ffbce9a7ca